mygame/app/species.rb

```diff
@@ -24,23 +24,596 @@
   source_y: 4
 ).freeze
 
+GREEN_BACK_SPRITE = EMOJI_BACK_BASE_SPRITE.merge(
+  source_x: 32,
+  source_y: 4
+).freeze
+
+PURPLE_BACK_SPRITE = EMOJI_BACK_BASE_SPRITE.merge(
+  source_x: 48,
+  source_y: 4
+).freeze
+
+BLUE_BACK_SPRITE = EMOJI_BACK_BASE_SPRITE.merge(
+  source_x: 80,
+  source_y: 4
+).freeze
+
 SPECIES = {
-  wink: {
-    name: 'WINK',
-    types: [:sassy, :sexy],
+  slightly_smiling: {
+    name: "Slightly Smiling",
+    sprite: EMOJI_BASE_SPRITE.merge(
+      tile_x: 0,
+      tile_y: 0
+    ),
+    back_sprite: YELLOW_BACK_SPRITE,
+    type: :silly
```

Comment:
I liked potential double types ;) as in the original .... should we leave them out for now?